components/pricing_modal/content.tsx

```diff
@@ -0,0 +1,346 @@
+            style={{
+                backgroundColor: props.bgColor,
+                color: props.color,
+            }}
+        >
+            {props.firstSvg}
+            {props.text}
+            {props.secondSvg}
+        </div>
+    );
+}
+
+function Card(props: CardProps) {
+    return (
+        <div className='PlanCard'>
+            {props.planLabel && props.planLabel}
+            <div
+                className='top'
+                style={{backgroundColor: props.topColor}}
+            />
+            <div className='bottom'>
+                <div className='plan_price_rate_section'>
+                    <h4>{props.plan}</h4>
+                    <h1 className={props.plan === 'Enterprise' ? 'enterprise_price' : ''}>{props.price}</h1>
+                    <p>{props.rate}</p>
+                </div>
+                <div className='plan_briefing'>
+                    <h4>{props.briefing.title}</h4>
+                    {props.briefing.items.map((i) => {
+                        return (
+                            <div
+                                className='item'
+                                key={i}
+                            >
+                                <i className='fa fa-circle bullet'/><p>{i}</p>
+                            </div>
+                        );
+                    })}
+                    {props.planDisclaimer && (
+                        <p className='disclaimer'><i className='icon-alert-outline'/>{props.planDisclaimer}</p>
+                    )}
+                </div>
+                <div>
+                    <button
+                        className={'plan_action_btn ' + props.buttonDetails.customClass}
+                        disabled={props.buttonDetails.disabled}
+                        onClick={props.buttonDetails.action}
+                    >{props.buttonDetails.text}</button>
+                </div>
+
+                {props.extraAction && (
+                    <button
+                        className='plan_action_btn'
+                        onClick={props.extraAction?.action}
+                    >{props.extraAction?.text}</button>
+                )}
+            </div>
+        </div>
+    );
+}
+
+function Content(props: ContentProps) {
+    const {formatMessage} = useIntl();
+    const dispatch = useDispatch();
+
+    const contactSalesLink = useSelector((state: GlobalState) => getCloudContactUsLink(state, InquiryType.Sales));
+
+    const subscription = useSelector(selectCloudSubscription);
+    const product = useSelector(selectCloudProduct);
+    const products = useSelector(selectCloudProducts);
+
+    const isEnterprise = product?.sku === CloudProducts.ENTERPRISE;
+    const professionalProduct = Object.values(products || {}).find(((product) => {
+        return product.sku === CloudProducts.PROFESSIONAL;
+    }));
+
+    let isStarter = false;
+    let isPostTrial = false;
+    if (product?.sku === CloudProducts.STARTER) {
+        isStarter = true;
+    }
```

Comment:
I believe we can simplify these conditions and the one below to:

```suggestion
    let isStarter = product?.sku === CloudProducts.STARTER;
    let isPostTrial = subscription?.trial_end_at && !subscription.is_free_trial;
```